src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
I'm not quite sure I'm following this. The current PR code is using the presence of `update_seq` as a proxy for "this is a response from an actual database" (as opposed to a response from a resource which returns a 200 but isn't a database at all).

@kxepal Is your suggestion that we should also check for the presence of `instance_start_time` in order to detect the case where a resource happens to provide an `update_seq` but no `instance_start_time`?